api/v1alpha1/crds/caren.nutanix.com_awsclusterconfigs.yaml

```diff
@@ -298,6 +298,23 @@ spec:
                         type: string
                     type: object
                 type: object
+              encryption:
+                description: |-
+                  Encryption defines the configuration to enable encryption at REST
+                  This configuration is used by API server to encrypt data before storing it in ETCD.
+                  Currently the encryption only enabled for secrets and configmaps.
+                properties:
+                  providers:
+                    default: aescbc
+                    description: Encryption providers
+                    enum:
+                    - aescbc
```

Comment:
Can we support kms V2 as well? Both these providers have issues that may not be acceptable to users.

Reply:
KMS V2 requires KMS configuration that we do not support at the moment. https://kubernetes.io/docs/tasks/administer-cluster/kms-provider/#encrypting-your-data-with-the-kms-provider-kms-v2

Reply:
Can we have a think how we can support it? Not a blocker for this pr but something we should support in future.

Reply:
Created https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/issues/624 to track it. We can add it to our JIRA backlog if it becomes requirement.